packages/documentation/src/stories/components/accordion/accordion-item.stories.ts

```diff
@@ -4,12 +4,26 @@ const meta: Meta = {
   id: '3b86bc9b-3dcd-4788-a436-88fd18a6312d',
   title: 'Components/Accordion Item',
   component: 'post-accordion-item',
+  parameters: {
+    controls: {
+      exclude: [
+        'accordion-item', // this CSS part is for internal use only
+      ],
+    },
+  },
   argTypes: {
     collapsed: {
-      control: false,
+      control: false, // disable the control since it is not usable on the story
     },
     headingLevel: {
-      control: false,
+      name: 'heading-level',
+      description:
+        'Defines the hierarchical level of the accordion item header within the headings structure.' +
+        '<span className="mb-mini alert alert-warning alert-sm">' +
+        '<code>heading-level</code> on <code>post-accordion-item</code> is deprecated. ' +
+        'Set it directly on the parent <code>post-accordion</code>.' +
+        '</span>',
```

Comment:
Can we move this description into the component file (as jsdoc block) or is there a problem with the alert?

Reply:
Good idea!
I had to add some [custom logic](https://github.com/swisspost/design-system/pull/3067/files#diff-3b36e8ee2b1769b10d210808ac598a870fe45bcc961438ce1f7e1e1af38f70beR33) but the deprecation message is now linked to the jsDocs